src/ui/LiveButton.js

```diff
@@ -9,23 +10,22 @@ import {
   UIText,
 } from '@apollosproject/ui-kit';
 import { AnalyticsConsumer } from '@apollosproject/ui-analytics';
-import { WebBrowserConsumer } from '../ui/WebBrowser';
-
-import { LiveConsumer } from '.';
+import {
+  RockAuthedWebBrowser,
+  LiveConsumer,
+} from '@apollosproject/ui-connected';
 
 const LiveCard = styled(({ theme }) => ({
   backgroundColor: theme.colors.lightSecondary,
 }))(Card);
 
-const LiveNowButton = () => (
-  <LiveConsumer>
-    {(liveStreams) => {
-      const isLive = !!liveStreams.length;
-
-      return isLive ? (
+const LiveNowButton = ({ contentId }) => (
+  <LiveConsumer contentId={contentId}>
```

Comment:
I mentioned this earlier, but this is the actual place I questioned. Not sure why we need to pass `contentId` to the `LiveConsumer` here ... we don't really care what the content is.

Reply:
same answer as before